scss/forms/_form-check.scss

```diff
@@ -113,9 +113,9 @@
 
     // Boosted mod
     &:checked,
-    &:indeterminate {
+    &[type="checkbox"]:indeterminate {
```

Comment:
Not sure to catch what's this use case here

Reply:
OK https://developer.mozilla.org/en-US/docs/Web/CSS/:indeterminate is indeed available for radio buttons too. So I understand better this selector.